src/test/isolation2/sql/fts_errors.sql

```diff
@@ -139,6 +139,33 @@ select wait_until_all_segments_synchronized();
 -- verify no segment is down after recovery
 select count(*) from gp_segment_configuration where status = 'd';
 
+-- start_ignore
+-- After error, schemas for temporary table still exist (like 'pg_temp'
+-- and 'pg_toast_temp').
+-- Lets remove all such temporary schemas for inactive connections
+DO $$
```

Comment:
How about avoid copy/paste using include like in https://github.com/arenadata/gpdb/blob/28e921ed8d68cc1754d6ddbdd300044e43066ec9/src/test/regress/sql/qp_with_functional_inlining.sql#L11

Reply:
It work only for regression tests.

Reply:
in isolation you may use psql -f like in
https://github.com/arenadata/gpdb/blob/28e921ed8d68cc1754d6ddbdd300044e43066ec9/src/test/isolation2/sql/resgroup/resgroup_cpuset_empty_default.sql#L21